clients/python/setup.py

```diff
@@ -5,12 +5,14 @@
 
 setuptools.setup(name='myanmartools',
     version='0.1',
+    url='https://github.com/google/myanmar-tools',
     author='William (Wai Yan) Zhu',
     author_email='williamzhu345@gmail.com',
+    license='Apache License, Version 2.0',
```

Comment:
Should we use the identifier string "Apache-2.0" here?  It doesn't seem very well documented…

Reply:
I think this license metadata is free text (you can give anything you want, like description and platforms). The best reference I can give is [this answer](https://stackoverflow.com/a/34994245/6293961). I used the name from [the official Apache page](https://www.apache.org/licenses/LICENSE-2.0) (nltk also [uses](https://github.com/nltk/nltk/blob/develop/setup.py) this name), but you can change it to anything you want.

Reply:
Thanks!